test/mocks/providers/provider_homepage.dart

```diff
@@ -44,10 +44,12 @@ final loadingHomePageProvider = ProviderScope(
 
 ProviderScope homePageFakeFirestoreProvider(
   FakeFirebaseFirestore firestore,
-  MockGeolocationService geoLocationService,
-) {
+  MockGeolocationService geoLocationService, {
+  List<Override> additionalOverrides = const [],
+}) {
   return ProviderScope(
     overrides: [
+      ...additionalOverrides,
```

Comment:
I think this is a good idea to add a `additionalOverrides`, but at the same time, I think it breaks the idea that I have of the mocks providers that we have (idea of ready to use). 

I understand that it adds some flexibility to our mocks providers. However, to keep our mock provider file consistent, shouldn't we duplicate this part and create a provider that directly adds the override that you use in your tests?

As for the moment, your additional override has a single usage, but if we want to expand your tests or use the same mock provider, we may have to duplicate the `additionalOverrides`.

I understand both sides: flexibility / duplication of providers.

I am curious to hear your thoughts about it 😄

Reply:
This is a very good point, I did not consider it at all. I completely agree with you, I'm just afraid it may decrease slightly the readability of the code. Therefore, please tell me what you think of the variable names I used in fe04e87d9d49d38286c9dcd78357133a4fc53a6d.